### DISCUSSION TITLE
Plotting arrows between cells of different tables

### DISCUSSION THREAD

**Discussion by psads-git** [cetz-package/cetz#645] [Q&A]
Can the following diagram be done in `cetz`, by specifying what are the cells to be connected by arrows?

I have already found a solution, by using `pinit` and a hacked function of it:

[Connecting tables cells with arrows](https://github.com/typst/typst/discussions/4510#discussioncomment-9978942)

I am wondering whether a solution for this problem can also be found using `cetz`.

![image](https://github.com/cetz-package/cetz/assets/75945439/a69961e6-f485-47ac-bf2c-806fd3a0bb31)

**Comment by johannes-wolf** [comment]
There is an open issue #607 for this: referencing things outsides a `canvas`. With the `floating` function, it is possible to draw elements without resizing the canvas. The missing bit is computing the position of an anchor outsides the canvas. I *think* we can do it like the `pinit` package does it, but I have not yet tested it.

**Comment by psads-git** [comment]
Thanks, @johannes-wolf, for your reply. That would be a great `cetz` development!
